packages/ts/react-crud/src/crud.d.ts

```diff
@@ -3,14 +3,15 @@ import type Pageable from './types/com/vaadin/hilla/mappedtypes/Pageable';
 
 export interface FormService<T> {
   save(value: T): Promise<T | undefined>;
+
   delete(id: any): Promise<void>;
 }
 
 export interface ListService<T> {
   list(request: Pageable, filter: FilterUnion | undefined): Promise<T[]>;
 }
 
-export interface CountService<T> {
+export interface CountService {
```

Comment:
This could be a significant change; originally I changed it because Eslint shown an error here (`T` is not used anywhere). I would like to have someone's opinion here.

Reply:
Could we change it to `T = unknown` or `T = never`?

Reply:
Well... That won't fix the Eslint issue because `T` is still not used anywhere in the body of the interface.